Kudu.Core/Deployment/DeploymentManager.cs

```diff
@@ -274,6 +274,8 @@ public void Delete(string id)
                 {
                     throw new DeploymentFailedException(exception);
                 }
+
+                return GetResult(id);
```

Comment:
Is this safe?

Reply:
Sorry for the delay, I'll get to this soon...

Reply:
Bump! Don't wanna nag, but thought I'd remind you about this in case it's been forgotten ⏳

Reply:
Thanks @nikolaia, have been working on other priorities but will get to this as soon as I can, it's still on my radar.

Reply:
Any updates here? :)